AllAboutGraphQL/Blog.MD

```diff
@@ -1,55 +1,47 @@
 
+There is no limitation in the GraphQL spec that enforces that the data source should be a graph.
 
 ## Myth: GraphQL is a database technology. 
-It's a misconception that you need to rewrite your database to adopt GraphQL. GraphQL is a wrapper for database. GraphQL is a query language for  API - which means it is a syntax of how to ask for data. 
-
+It's a misconception that you need to rewrite your database to adopt GraphQL. GraphQL can be a wrapper around any data source, including databases. GraphQL is a `query language for your API` - which means it is a syntax of how to ask for data. 
 
 ## Myth: Resolvers/interfaces/queries/mutations work magically. 
-You will need to define exactly what each of them needs to do. You will be writing functions that gets called when queries are fired, writing functions for resolvers which sends back exactly the data you need and knows which API to call. Things don't automatically start working when you fire a query or mutation. You will be defining what data returns through those functions by calling resolvers. 
-
-
-## Myth GraphQL replaces Redux / state management library
-Redux is a state management library. GraphQL is not a state management library. GraphQL helps to get less data, which in turn leads to less data to manage on the client side, but it is not a state management solution. You will still need to manage state - albeit lightweight. Client libraries like Apollo have cache built-in which can be used to manage state. GraphQL is not a replacement for Redux - it helps to reduce the need for it.  
-
-
-## Myth GraphQL is a database language. 
-GraphQL is a programming language - specifically a query language. GraphQL's spec defines how GraphQL engines should implement the language, how data should be communicated between client and server. 
+You will need to define exactly what each of them needs to do. You will be writing functions that get called when queries are fired, writing functions for resolvers that send back exactly the data you need and knows which API to call. You will be defining what data returns through those functions by calling resolvers. 
 
+## Myth: GraphQL replaces Redux or any state management library
+Redux is a state management library. GraphQL is not a state management library. GraphQL helps to get less data, which in turn leads to less data to manage on the client-side, but it is not a state management solution. You will still need to manage state - albeit lightweight. Client libraries like Apollo have cache built-in which can be used to manage state. GraphQL is not a replacement for Redux - it helps to reduce the need for it.  
 
-## Myth You can't have REST endpoints in your implementation with GraphQL. 
-You can plugin multiple REST endpoints or even multiple GraphQL endpoints in your application. Although it is not a best practice to have multiple REST endpoints, it is technically possible. 
+## Myth: GraphQL is a database language. 
```

Comment:
There are some DBs that use GraphQL as a query language, e.g. [FaunaDB](https://fauna.com/faunadb) and [dGraph](https://dgraph.io/)